src/components/Player/Player.scss

```diff
@@ -0,0 +1,9 @@
+.player {
```

Comment:
I do not see this class being used in Player.html

Reply:
https://screencast.com/t/ACzzoY1ZV 
Should I put ".player" to the div I added or to the div I attach to?

Reply:
Does it mean that <div> inside main can have ".main__player" class and than I mount <div> with ".player"?

Reply:
Yes inside main you should use .main__player. 

what do you mean by ` than I mount with ".player"?`